enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -150,3 +150,9 @@ def get_product_source(self, enterprise_customer_uuid, content_identifier):
             content_identifier
         )
         return self.product_source_for_content(course_details)
+
```

Comment:
nit: Consider caching - doesn't have to block, but we should make sure there's a ticket in the backlog to eventually cache all this stuff at the right level of granularity.

Reply:
agreed. something i'll add into a later iteration.